### PR TITLE
feat(error-orchestrator): Improved on createGetterSetter

### PR DESCRIPTION
**Pull request by gabiwassan**
Hi team,
This PR applies the ErrorOrchestratorService enhancement by applying createGetterSetter.
ex: 

```
try {
    ...........
} catch (error) {
    const options = {
      level: 'INFO',
      severity: 'CRITICAL',
      error: error;
    };

    getErrorOrchestrator().handleError(options);
}
```

### CODE REVIEW

**Review by Machi3mfl — APPROVED** [review]
LGTM!

**Review by pablomarga — APPROVED** [review]
LGTM!

File: public/kibana-services.ts [hunk 1]
```diff
@@ -40,9 +40,9 @@ export const [getVisualizationsPlugin, setVisualizationsPlugin] = createGetterSe
 export const [getNavigationPlugin, setNavigationPlugin] = createGetterSetter<
   NavigationPublicPluginStart
 >('NavigationPlugin');
-export const [getErrorOrchestrator, setErrorOrchestrator] = createGetterSetter<ErrorOrchestrator>(
-  'ErrorOrchestrator'
-);
+export const [getErrorOrchestrator, setErrorOrchestrator] = createGetterSetter<
+  ErrorOrchestratorService
+>('ErrorOrchestratorService');
```
Comment (Desvelao) [inline public/kibana-services.ts:45]: I don't undestand what is the reason to use a `createGetterSetter` for this.

Comment (gabiwassan) [reply public/kibana-services.ts:45]: The idea is that we can use the orchestrator throughout the app without having to create a new instance every time, like the other services (toast, https, plugins, etc)

Comment (frankeros) [reply public/kibana-services.ts:45]: I think that is a good approach to get the common services from one place, you need to only know from where import `kibana-services` (maybe it need a new name)

Comment (frankeros) [reply public/kibana-services.ts:45]: Also, maybe not for this case where the instance is static but for another service that needs a constructor for its instance is good to get the same instance from a getterSetter

**Review by frankeros — APPROVED** [review]
LGTM!

**Review by asteriscos — APPROVED** [review]
CR: :heavy_check_mark:

**Pull request merged**